### ISSUE TITLE
upgrade to Filament 3

### ISSUE THREAD

**Issue by marco76tv** [Sevendays-Digital/filament-nested-resources#18]

**Comment by marco76tv** [comment]
when can i use it with filament 3 ?

**Comment by haringsrob** [comment]
Not sure, if you need it urgently you can always make a pr.

**Comment by marco76tv** [comment]
Yes I'm working on it, any help is welcome, as soon as it's stable I'll make the pull request

https://github.com/laraxot/filament-nested-resources

**Comment by haringsrob** [comment]
will test it out then 👍

**Comment by tognit** [comment]
What is the status for filament version 3?
